testdata/templates/args.yaml

```diff
@@ -0,0 +1,8 @@
+service: foo
+headers:
+  header1: ${user:foo}
```

Comment:
Note: this is not replaced at the moment, template arguments are used for request bodies.

Before 1.0, we may want to think about processing everything in the YAML file? We would probably decode the template to a `map`, use `templateargs.ProcessMap`, then pass it through `mapdecode`. Thoughts @kriskowal @abhinav